server/server.go

```diff
@@ -88,7 +88,9 @@ func main() {
 	logger.Info("Started.", struct {
 		BackendId string `json:"backendId"`
 	}{backendId})
-	if task, ok := tasks[config.Task]; ok {
+	if config.Worker {
```

Comment:
Instead of adding a new parameter, you could just create a task with the name `worker` as it is the same kind of handler. So it could be used like `./server --task worker`